src/common/vitals/cumulative-layout-shift.js

```diff
@@ -0,0 +1,13 @@
+import { onCLS } from 'web-vitals'
+import { VITAL_NAMES } from './constants'
+import { VitalMetric } from './vital-metric'
+import { isBrowserScope } from '../constants/runtime'
+
+export const cumulativeLayoutShift = new VitalMetric(VITAL_NAMES.CUMULATIVE_LAYOUT_SHIFT, (x) => x)
+
+if (isBrowserScope) {
+  onCLS(({ value, entries }) => {
+    if (value === cumulativeLayoutShift.value.current) return
```

Comment:
If the purpose is to _not_ have adjacent (reported) values be the same, note that the value in `.value.current` has been potentially rounded, so this comparison may return a false negative.

Reply:
b6bf3b1af9b4eb9c8788f9e59cfa0855e49499db